internal/provider/resource_access_flow.go

```diff
@@ -428,4 +380,11 @@ func (a *accessFlowResource) ValidateConfig(ctx context.Context, req resource.Va
 			"must be -1 or positive number",
 		)
 	}
+
+	if len(model.IntegrationTargets) == 0 && len(model.BundleTargets) == 0 {
```

Comment:
did we have validation for integration targets prior to this PR?

Reply:
before, the `integration_targets` field was required so basically yes